### DISCUSSION TITLE
the input control can not input Chinese

### DISCUSSION THREAD

**Discussion by Vjze** [marc2332/freya#615] [Q&A]
Using Freya for a few days, I feel good, but I also found some minor problems, the input control can not input Chinese, in addition, there seems to be no date picker and time picker control, I don't know if this control is within the developer's development mileage, English is not good, forgive me. Thanks again to the developers for their hard work and to give Rust development one more GUI to choose from.

**Comment by marc2332** [comment]
Yes, chinese is not supported currently -> https://github.com/marc2332/freya/issues/494

There isn't a date/time picker components yet, but it's definitely something that should be added

**Reply by Vjze** [reply]
Excuse me, my software interface has two input boxes, I need to click to jump to another input box, I don't know if I can use use_focus().

**Comment by marc2332** [comment]
Closed in https://github.com/marc2332/freya/pull/622
